client/utils.js

```diff
@@ -70,9 +72,9 @@ export const useWindowSize = () => {
    * @returns {Promise->Object}: promise which resolves to object
    */
 export const fetchGamePhoto = (gameid) => {
-  if (!gameid || gameid > 100 || gameid < 1) {
-    throw new Error('Invalid game id');
-  }
+  // if (!gameid || gameid > 100 || gameid < 1) {
```

Comment:
Rather than commenting these out entirely, consider upping the limit to 10000000.